pkgs/build-support/testers/hasPkgConfigModules/tester.nix

```diff
@@ -1,10 +1,20 @@
 # Static arguments
 { lib, runCommand, pkg-config }:
 
+let
+  genTestName = moduleNames:
+    # An overly long name made the progress bar weird and distraction, and same
+    # for the build log prefix.
+    if lib.length moduleNames > 1 then
+      "modules"
```

Comment:
I guess in this case at least add `package.pname` in there so it's possible to figure out where a failure comes from.

Reply:
Good idea. I guess actually `package.pname or package.name` is all we need.
Simplifies things.